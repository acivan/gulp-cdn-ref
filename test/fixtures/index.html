<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Document</title>
    <link rel="stylesheet" href="style/another.css">
</head>
<body>
    <img src="/img/inline.png" alt="">

    <script src="script/main.js"></script>
</body>
</html>