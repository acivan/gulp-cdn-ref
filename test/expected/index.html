<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Document</title>
    <link rel="stylesheet" href="http://lisposter.b0.upaiyun.com/style/another.css">
</head>
<body>
    <img src="http://lisposter.b0.upaiyun.com/img/inline.png" alt="">

    <script src="http://lisposter.b0.upaiyun.com/script/main.js"></script>
</body>
</html>